cmd/clusterctl/client/cluster/objectgraph.go

```diff
@@ -110,6 +114,10 @@ func (n *node) isSoftOwnedBy(other *node) bool {
 	return ok
 }
 
+func (n *node) getFilename() string {
+	return n.identity.Kind + "_" + n.identity.Name + "_" + n.identity.Namespace + ".yaml"
```

Comment:
I don't have a strong opinion on that but I would prefer `<kind>_<namespace>_<name>` (then it's "sorted" by namespace on a regular file listing)

Reply:
Sounds reasonable to me!